docs/data/material/components/snackbars/snackbars.md

```diff
@@ -65,6 +65,23 @@ You can use the `TransitionComponent` prop to change the transition of the Snack
 
 {{"demo": "TransitionsSnackbar.js"}}
 
+## Events
```

Comment:
I think this should be moved under `Customization` with title "Preventing default click away event".

Reply:
Updated https://deploy-preview-45629--material-ui.netlify.app/material-ui/react-snackbar/#preventing-default-click-away-event